deployments/memory-registry.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: memory-registry
+  labels:
+    app: memory-registry
+spec:
+  selector:
+    matchLabels:
+      app: memory-registry
+  template:
+    metadata:
+      labels:
+        app: memory-registry
+    spec:
+      containers:
+        - image: docker.pkg.github.com/networkservicemesh/cmd-registry-memory/cmd-registry-memory:latest
+          imagePullPolicy: IfNotPresent
+          name: memory-registry
+          env:
+            - name: SPIFFE_ENDPOINT_SOCKET
+              value: unix://run/spire/sockets/agent.sock
+          volumeMounts:
+            - name: spire-agent-socket
+              mountPath: /run/spire/sockets
+              readOnly: true
+      volumes:
```

Comment:
@edwarnicke, @haiodo In this PR I'd manually added volume with spire sockets into deployemnt. Should we consider using the device plugin/admission webhook to automatically appending this kind stuff for templates?

Reply:
@anvega What is the state of the art here?  Is there a nicer and cleaner way than using per-Pod-spect?